examples/services/streams.py

```diff
@@ -0,0 +1,39 @@
+import asyncio
+import logging
+from pathlib import Path
+
+from symphony.bdk.core.config.loader import BdkConfigLoader
+from symphony.bdk.core.symphony_bdk import SymphonyBdk
+from symphony.bdk.gen.pod_model.stream_filter import StreamFilter
+from symphony.bdk.gen.pod_model.stream_type import StreamType
+from symphony.bdk.gen.pod_model.v3_room_attributes import V3RoomAttributes
+
+
+async def run():
+    stream_id = "ubaSiuUsc_j-_lVQ8vhAz3___opSJdJZdA"
```

Comment:
maybe we can give a hint on how a stream id can be retrieved with a comment? (i.e click on a timestamp in a conversation in the client)